fpix/inc/ParseGuide.h

```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 //
 #include <map>
+#include <string>
+#include <stdint.h>
```

Comment:
```suggestion
#include <cstdint>
```

Reply:
`stdint.h` is part of the C standard library. `cstdint` is the C++ equivalent.

Reply:
I am using GCC 10.2.1 on Debian Bullseye.